backend/app/adapter/gqlapi/resolver/authmutation_test.go

```diff
@@ -0,0 +1,235 @@
+// +build !integration all
+
+package resolver
+
+import (
+	"testing"
+	"time"
+
+	"github.com/short-d/app/fw/assert"
+	"github.com/short-d/app/fw/crypto"
+	"github.com/short-d/app/fw/timer"
+	"github.com/short-d/short/backend/app/entity"
+	"github.com/short-d/short/backend/app/usecase/authenticator"
+	"github.com/short-d/short/backend/app/usecase/changelog"
+	"github.com/short-d/short/backend/app/usecase/keygen"
+	"github.com/short-d/short/backend/app/usecase/repository"
+	"github.com/short-d/short/backend/app/usecase/risk"
+	"github.com/short-d/short/backend/app/usecase/url"
+	"github.com/short-d/short/backend/app/usecase/validator"
+)
+
+func TestUpdateURL(t *testing.T) {
+	now := time.Now().UTC()
+	newAlias := "myNewAlias"
+	newLongLink := "https://www.short-d.com"
+	urls := urlMap{
+		"mySimpleAlias": entity.ShortLink{
```

Comment:
Recommend removing `my` so that every test case looks unbiased in a shared codebase.

Reply:
Thank you will remove it from the test cases!